tests/flytekit/unit/core/functools/test_decorator_location.py

```diff
@@ -0,0 +1,11 @@
+import importlib
+
+from flytekit import task
+from flytekit.core.tracker import extract_task_module
+
+
+def test_dont_use_wrapper_location():
+    m = importlib.import_module("tests.flytekit.unit.core.functools.decorator_usage")
+    get_data_task = getattr(m, "get_data")
+    assert "decorator_source" not in get_data_task.name
+    assert "decorator_usage" in get_data_task.name
```

Comment:
do you think it's worth having a more explicit unit test that exercises `extract_task_module`?

Reply:
probably

Reply:
wait no... this does do that already.

Reply:
it does that indirectly, right? I was talking about explicit calls to that function.